runtime/doc/builtin.txt

```diff
@@ -3170,7 +3209,8 @@ getmatches([{win}])					*getmatches()*
 		as |setmatches()| can restore a list of matches saved by
 		|getmatches()|.
 		If {win} is specified, use the window with this number or
-		window ID instead of the current window.
+		window ID instead of the current window.  If {win} is invalid,
+		an empty list is returned.
```

Comment:
actually seems to return 0 for nvim, probably a minor inaccuracy with the port of v8.1.1084

Reply:
_and_ an error, for good measure 😄